docs/contributor/layered_product_ci.md

```diff
@@ -0,0 +1,191 @@
+# Layered Product Testing with MicroShift
+
+A Layered Product is a software package developed independently from the MicroShift
+source repository and installed as a dependency atop MicroShift. For example, see
+[MicroShift GitOps](https://docs.redhat.com/en/documentation/red_hat_build_of_microshift/4.16/html/installing/microshift-install-rpm#microshift-installing-rpms-for-gitops_microshift-install-rpm).
+
+This document outlines a technique for setting up runtime environments using
+bootable containers to facilitate testing of such software packages. These tests
+can be conducted manually or integrated into the package CI/CD processes. See
+[Image Mode for MicroShift](./image_mode.md) for more information.
+
+> MicroShift includes comprehensive [Automated Functional and Integration Tests](../../test/README.md)
+> for features developed within its source repository.
+
+## Build and Publish MicroShift Container Image
+
+Follow the instructions in the [Build Image](./image_mode.md#build-image) section
+to implement the MicroShift container image layer build procedure.
+
+> Prebuilt MicroShift bootc container images are not currently available for
+> download.
```

Comment:
Can we fix this, at least for internal use? 
It cant be that hard to find a private registry/repo where we could push our image to. E.g. quay.io can be used for that using employee accounts. I would really make it as simple as possible for layered product teams. Each step we can elimnate lowers the bar to adoption.

Reply:
We can consider this, but it's not part of this task / PR.
As soon as we have something, we can change the wording appropriately.